### PR TITLE
Made some edits to the Miami-311 output and created Miami building permits service

### PR DESCRIPTION
**Pull request by yamilethmedina**

### CODE REVIEW

**Comment by bigfleet** [comment]
@yamilethmedina Nice!  Not many cities add second endpoints.  Kudos Miami.

I got this error when testing it out, see if you can duplicate:

TypeError at /miami-building-permits
no implicit conversion of String into Integer
file: miami-building-permits.rb location: [] line: 24

**Comment by yamilethmedina** [comment]
@bigfleet Can you point me to some documentation as to how I can test this? citygram-services itself isn't running for me in the console. I made a potential bug fix based on my research, but i'm not sure whether it works until I can test it. I'm not sure whether my IDE (Cloud9) is the problem or not.

**Comment by yamilethmedina** [comment]
Fixed! The endpoint is now working

File: lib/spy_glass/registry/miami-building-permits.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+require 'spy_glass/registry'
+require 'pp'
```
Comment (bkerley) [inline lib/spy_glass/registry/miami-building-permits.rb]: oh heh, forgot to take the `pp` stuff out

**Comment by bkerley** [comment]
Probably don't want to merge your `.env.swp` file

**Comment by yamilethmedina** [comment]
@bigfleet @lyzidiamond I believe this is ready to push to the master branch. Can I get a second look at it?

**Comment by bigfleet** [comment]
@yamilethmedina @bkerley I switched ID (which is required) to permit number.  If permit numbers are not unique, please submit a subsequent PR as it'll be a bad user experience when duplication occurs.  But congratulations, this will hit production today!

**Comment by bkerley** [comment]
They *should* be unique, but in a few minutes I'll check for any duplication in 2014, 2015, and 2016 (if @socrata supports those queries :) ).

**Comment by bkerley** [comment]
If [this query](https://opendata.miamidade.gov/Building/duplicate-permit-numbers/f8b4-pv4f) does what I think it does (\*impotently shakes fist at visual query builders\*), there are no duplicates, and we're gravy. Thanks for doing most of the work, @yamilethmedina , and thanks for your help, @bigfleet !

**Comment by yamilethmedina** [comment]
I'm so happy about this! Couldn't have done it without you @bkerley !

On Thu, Mar 10, 2016 at 9:25 AM, bkerley <notifications@github.com> wrote:

> If this query
> <https://opendata.miamidade.gov/Building/duplicate-permit-numbers/f8b4-pv4f>
> does what I think it does (*impotently shakes fist at visual query
> builders*), there are no duplicates, and we're gravy. Thanks for doing most
> of the work, @yamilethmedina <https://github.com/yamilethmedina> , and
> thanks for your help, @bigfleet <https://github.com/bigfleet> !
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citygram/citygram-services/pull/53#issuecomment-194870709>
> .
>



-- 
*Yami Medina*
*Web Developer: JavaScript, Ruby on Rails, HTML, CSS, and more!*
https://www.github.com/yamilethmedina
https://www.linkedin.com/in/yamilethmedina

**Pull request merged**